sklearn/cluster/dbscan_.py

```diff
@@ -74,6 +74,13 @@ def dbscan(X, eps=0.5, min_samples=5, metric='minkowski',
     n_jobs : int, optional (default = 1)
         The number of parallel jobs to run for neighbors search.
         If ``-1``, then the number of jobs is set to the number of CPU cores.
+        Parrallelism currently only used when ``algorithm="brute"``, where
```

Comment:
I'm really not finding this clear and succinct enough, which is why I suggested a specific wording.